eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/AsyncHttpRequestManager.java

```diff
@@ -18,49 +18,52 @@
  * asynchronous requests and multiple threads.
  *
  * @author HMS Networks, MU Americas Solution Center
- * @version 2.0.0
+ * @version 1.0.0
+ * @since 2.0.0
```

Comment:
Version is less than since?

Reply:
Since identifies the application version 'since' the class was added. Version identifies the version of the class itself.

I'm not sure how, but I misconfigured the tags in the original commit that added this class.

Reply:
I was getting confused by the .pom for this. I assume the .pom will be updated to 2.0.0 in a coming PR?

Reply:
I think if you add version references and dont release that version in the PR there should be a note that it is coming so as to not confuse reviewers. I dont seee anything that says that it couldnt be 1.3.0.

Reply:
Yes, there will be another PR later this week. That is a good point, though. I can add a note.